src/Main.java

```diff
@@ -0,0 +1,45 @@
+// Press Shift twice to open the Search Everywhere dialog and type `show whitespaces`,
+// then press Enter. You can now see whitespace characters in your code.
+public class Main {
+    public static void main(String[] args) {
+        // Press Opt+Enter with your caret at the highlighted text to see how
+        // IntelliJ IDEA suggests fixing it.
+        System.out.printf("Hello and welcome!");
+
+
+        int positiveOrNegativeOrZero = 0;
+        if (positiveOrNegativeOrZero > 0) {
+            System.out.println("It's positive");
+        } else if (positiveOrNegativeOrZero < 0) {
+            System.out.println("It's negative");
+        } else if (positiveOrNegativeOrZero == 0){
+            System.out.println("It's a zero");
+        }
+
+
+String bartender = "wine";
+
+        switch (bartender) {
```

Comment:
Je hebt een leuke oplossing bedacht en je maakt goed gebruik van de doorval functie van de switch. Het staat alleen wederom niet in een "bartender" methode. Ook heb je geen default case. Het is bij een switch belangrijk om altijd een default case te maken.